onnxruntime/test/providers/cpu/math/matmul_test.cc

```diff
@@ -269,10 +396,16 @@ void RunMatMulZeroKTest() {
 }
 
 TEST(MathOpTest, MatMulZeroKFloatType) {
+  if (DefaultWebGpuExecutionProvider().get() != nullptr) {
+    GTEST_SKIP() << "Skipping because of the following error: Webgpu does not support zero-sized tensor";
```

Comment:
maybe add the WebGPU EP here instead, for consistency.

https://github.com/microsoft/onnxruntime/blob/6de4d8a9b6974479cf47028b45e33a52ab449792/onnxruntime/test/providers/cpu/math/matmul_test.cc#L390-L393